examples/tutorials/workgraphs/descriptors_filter_qe.ipynb

```diff
@@ -0,0 +1,533 @@
+    "            'prepend_text': '''\n",
+    "            module purge\n",
+    "            module use /work4/scd/scarf562/eb-common/modules/all\n",
+    "            module load amd-modules\n",
+    "            module load QuantumESPRESSO/7.2-foss-2023a\n",
+    "            ''',\n",
+    "            'append_text': ''  \n",
+    "        }\n",
+    "    }\n",
+    "\n",
+    "    pseudo_family = load_group('SSSP/1.3/PBE/efficiency')\n",
+    "\n",
+    "    for _, file in inputs.items():\n",
+    "        with file.as_path() as path:\n",
+    "            for i, structs in enumerate(iread(path, format=\"extxyz\")):\n",
+    "                \n",
+    "                structure = StructureData(ase=structs)\n",
+    "                pseudos = pseudo_family.get_pseudos(structure=structure)\n",
+    "\n",
+    "                ecutwfc, ecutrho = pseudo_family.get_recommended_cutoffs(\n",
+    "                    structure=structure,\n",
+    "                    unit='Ry',\n",
+    "                )\n",
+    "\n",
+    "                pw_paras = {\n",
+    "                    \"CONTROL\": {\n",
+    "                        \"calculation\": \"scf\",\n",
+    "                        'tprnfor': True,\n",
+    "                        'tstress': True,\n",
+    "                    },\n",
+    "                    \"SYSTEM\": {\n",
+    "                        \"ecutwfc\": ecutwfc,\n",
+    "                        \"ecutrho\": ecutrho,\n",
+    "                    },\n",
+    "                }\n",
+    "\n",
+    "                qe_task = wg.add_task(\n",
+    "                    PwCalculation,\n",
+    "                    code = qe_code,\n",
+    "                    parameters= pw_paras,\n",
+    "                    kpoints= kpoints,\n",
+    "                    pseudos= pseudos,\n",
+    "                    metadata= metadata,\n",
+    "                    structure= structure,\n",
+    "                )\n",
+    "\n",
+    "                structfile = f\"{_}.struct{i}\"\n",
+    "                stuctparam = f\"{_}.struct{i}_param\"\n",
+    "                wg.update_ctx({structfile: qe_task.outputs.output_trajectory})\n",
+    "                wg.update_ctx({stuctparam: qe_task.outputs.output_parameters})\n",
+    "                \n",
+    "    print(wg.tasks)\n",
+    "\n",
+    "    return {\n",
+    "        \"test_file\": wg.ctx.test_file,\n",
+    "        \"train_file\": wg.ctx.train_file,\n",
+    "        \"valid_file\": wg.ctx.valid_file\n",
+    "    }    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "695c1d37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida_mlip.data.config import JanusConfigfile\n",
+    "from aiida.orm import Dict, TrajectoryData\n",
+    "from ase.io import write\n",
+    "@task.calcfunction(outputs = [\"JanusConfigfile\"])\n",
+    "def create_train_file(**inputs):\n",
+    "\n",
+    "    GPa_to_eV_per_A3 = 1.0 / 160.2176621\n",
```

Comment:
Where did you get this from? There should be something equivalent that can be imported from `ase.units`.